.github/workflows/sign-darwin.yml

```diff
@@ -0,0 +1,61 @@
+---
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'release tag to sign'
+        required: false
+        type: string
+
+jobs:
+  sign-binary:
+    runs-on: macos-latest
+    steps:
+      - uses: dsaltares/fetch-gh-release-asset@1.0.0
+        with:
+          version: "tags/${{ inputs.tag }}"
+          file: "konvoy-image-bundle-${{ inputs.tag }}_darwin_amd64.tar.gz"
+
+      - name: Extract the pulled file
+        run: tar -xvzf "konvoy-image-bundle-${{ inputs.tag }}_darwin_amd64.tar.gz" -C "konvoy-image-bundle_darwin_amd64"
+
+      - name: Remove original downloaded file
+        run: rm "konvoy-image-bundle-${{ inputs.tag }}_darwin_amd64.tar.gz"
+
+      - name: Fetch current main directory
+        uses: actions/checkout@v3
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - name: Import Code-Signing Certificates
+        uses: Apple-Actions/import-codesign-certs@v1
+        with:
+          # The certificates in a PKCS12 file encoded as a base64 string
+          p12-file-base64: ${{ secrets.APPLE_DEVELOPER_CERTIFICATE_P12_BASE64 }}
+          # The password used to import the PKCS12 file.
+          p12-password: ${{ secrets.APPLE_DEVELOPER_CERTIFICATE_PASSWORD }}
+
+      - name: Install required binaries
+        run: |
+          brew tap mitchellh/gon
+          brew install mitchellh/gon/gon
+
+      - name: Run Gon to notarize
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          AC_USERNAME: ${{ secrets.AC_USERNAME }}
+          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+        run: gon ./gon.hcl -log-level=info -log-json ./config.hcl
```

Comment:
where is config.hcl coming from?

Reply:
I believe this is a file created for the logs: https://github.com/mitchellh/gon/blob/master/README.md